reflex/.templates/web/components/shiki/code.js

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react"
+import { codeToHtml} from "shiki"
+
+export function Code ({code, theme, language, transformers, ...divProps}) {
+    const [codeResult, setCodeResult] = useState("")
+    useEffect(() => {
+        async function fetchCode() {
+          let final_code;
+
+          if (Array.isArray(code)) {
+            final_code = code[0];
+          } else {
+            final_code = code;
+          }
```

Comment:
This is more of a hack to get reflex web to run. somehow, a list is sometimes passed as the value of `code`. Have a tracking ticket to look into that https://linear.app/reflex-dev/issue/ENG-3893/get-shiki-to-work-with-reflex-web